src/orchestrator-kubernetes/src/lib.rs

```diff
@@ -790,6 +792,18 @@ impl NamespacedOrchestrator for NamespacedKubernetesOrchestrator {
             None
         };
 
+        let security_context = if let Some(fs_group) = self.config.service_fs_group {
+            Some(PodSecurityContext {
+                // clusterd and environmentd both always run as gid 999, as defined
+                // in their `Dockerfile`'s. This change is only relevant when there is
+                // a volume mount, but we apply is unconditionally.
```

Comment:
```suggestion
```

Comments should follow the abstraction boundary too! Also the bit about unconditionally is no longer true. So best to just remove I think.

Reply:
will remove!